A6-client-retries.md

```diff
@@ -177,6 +192,12 @@ Neither retry attempts or hedged RPCs block when `token_count` is less than or e
 
 The only RPCs that are counted as failures for the throttling policy are RPCs that fail with a status code that qualifies as a [retryable](#retryable-status-codes) or [non-fatal status code](#hedging-policy), or that receive a pushback response indicating not to retry. This avoids conflating server failure with responses to malformed requests (such as the `INVALID_ARGUMENT` status code).
 
+##### Validation of retryThrottling
+
+If `retryThrottling` is specified in a service config, the following validation rules apply:
```

Comment:
Similar question here.

Reply:
We would consider the entire service config invalid as per https://github.com/grpc/proposal/blob/master/A21-service-config-error-handling.md#criteria-to-determine-the-validity-of-a-service-config